pkg/operatorsource/configuring_test.go

```diff
@@ -122,48 +129,18 @@ func TestReconcile_CatalogSourceConfigAlreadyExists_Updated(t *testing.T) {
 		TargetNamespace: opsrcIn.Namespace,
 		Packages:        packages,
 	}
-	kubeclient.EXPECT().Update(context.TODO(), cscWant).Return(nil)
+
+	reconciler := operatorsource.NewConfiguringReconciler(helperGetContextLogger(), datastore, fakeclient)
 
 	opsrcGot, nextPhaseGot, errGot := reconciler.Reconcile(ctx, opsrcIn)
 
 	assert.NoError(t, errGot)
 	assert.Equal(t, opsrcIn, opsrcGot)
 	assert.Equal(t, nextPhaseWant, nextPhaseGot)
-}
 
-// Use Case: Update of existing CatalogSourceConfig object fails.
-// Expected Result: The object is moved to "Failed" phase.
-func TestReconcile_UpdateError_MovedToFailedPhase(t *testing.T) {
```

Comment:
@tkashem I have looked at this test again, and I don't see much added value by having it. The unique code path is within an error check that sets the `nextPhase` to `Failed` while other error checks aren't given their own tests. Let me know if you have any objections.